### ISSUE TITLE
Recordings not showing in web UI

### ISSUE THREAD

**Issue by mjmeli** [blakeblackshear/frigate#1830]
**Describe the bug**
I have recordings set up on one of my cameras and the recordings are being saved to the `/recordings` directory, but the web UI isn't picking up that they exist and there is no UI. Clips work totally fine.

**Version of frigate**
0.8.4-5043040

**Config file**
Include your full config file wrapped in triple back ticks.
```yaml
mqtt:
  host: 192.168.86.28
  user: frigateuser
  password: '{FRIGATE_MQTT_PASSWORD}'
detectors:
  coral:
    type: edgetpu
    device: usb
clips:
  retain:
    default: 30
ffmpeg:
  hwaccel_args:
    - -hwaccel
    - vaapi
    - -hwaccel_device
    - /dev/dri/renderD128
    - -hwaccel_output_format
    - yuv420p
cameras:
  front-door:
    rtmp:
      enabled: False
    clips:
      enabled: True
    snapshots:
      enabled: True
      bounding_box: True
      timestamp: True
    record:
      enabled: True
      retain_days: 30
    objects:
      track:
        - person
        - car
        - dog
      filters:
        person:
          mask:
            - 1008,657,1006,714,928,714,924,659
    ffmpeg:
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -r
        - "5" # <---- adjust depending on your desired frame rate from the mjpeg image
        #- -use_wallclock_as_timestamps
        #- "1"
      inputs:
        - path: xxxx
          roles:
            - detect
            - clips
            - record
    width: 1600
    height: 1200
    fps: 5
    zones:
      driveway:
        coordinates: 1052,849,1119,843,1120,773,1031,754,909,755

```

**Frigate container logs**
```
Starting migrations,
There is nothing to migrate,
peewee_migrate                 INFO    : Starting migrations,
 * Starting nginx nginx,
   ...done.,
peewee_migrate                 INFO    : There is nothing to migrate,
frigate.mqtt                   INFO    : MQTT connected,
frigate.app                    INFO    : Camera processor started for front-door: 37,
frigate.edgetpu                INFO    : TPU found,
detector.coral                 INFO    : Starting detection process: 34,
frigate.app                    INFO    : Capture process started for front-door: 38,
frigate.edgetpu                INFO    : Attempting to load TPU as usb
```

**Frigate stats**
```json
{
  "detection_fps": 7.4, 
  "detectors": {
    "coral": {
      "detection_start": 0.0, 
      "inference_speed": 10.68, 
      "pid": 34
    }
  }, 
  "front-door": {
    "camera_fps": 24.0, 
    "capture_pid": 38, 
    "detection_fps": 7.4, 
    "pid": 37, 
    "process_fps": 20.6, 
    "skipped_fps": 0.0
  }, 
  "service": {
    "storage": {
      "/dev/shm": {
        "free": 63.7, 
        "mount_type": "tmpfs", 
        "total": 67.1, 
        "used": 3.4
      }, 
      "/media/frigate/clips": {
        "free": 4863773.7, 
        "mount_type": "btrfs", 
        "total": 7676736.0, 
        "used": 2812962.3
      }, 
      "/media/frigate/recordings": {
        "free": 4863773.7, 
        "mount_type": "btrfs", 
        "total": 7676736.0, 
        "used": 2812962.3
      }, 
      "/tmp/cache": {
        "free": 997.0, 
        "mount_type": "tmpfs", 
        "total": 1000.0, 
        "used": 3.0
      }
    }, 
    "uptime": 60, 
    "version": "0.8.4-5043040"
  }
}
```

**Screenshots**
No recording links
![image](https://user-images.githubusercontent.com/6422129/134542952-23a47964-959b-4179-bbe0-c119f5c46dc1.png)

Recordings exist
![image](https://user-images.githubusercontent.com/6422129/134543105-2e9eb5bc-0ab1-4518-bee5-7d7f0abee80f.png)
![image](https://user-images.githubusercontent.com/6422129/134543124-84daafd7-04ba-47ef-a71d-2f7f8de450cf.png)
![image](https://user-images.githubusercontent.com/6422129/134543155-eb5ccfa8-305d-46bb-bf7a-852e33e194a7.png)

The Home Assistant integration also can't access recordings
![image](https://user-images.githubusercontent.com/6422129/134544823-7f809fc1-25ec-4a3c-99ab-cc2aec7193df.png)

**Computer Hardware**
Synology NAS
Docker Compose
```yaml
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: blakeblackshear/frigate:stable-amd64
    devices:
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    volumes:
      - /dev/bus/usb:/dev/bus/usb # for coral AI USB accelerator - only works as a volume mount not a device mount
      - /etc/localtime:/etc/localtime:ro 
      - /volume1/docker/frigate/volume/config/config.yml:/config/config.yml:ro
      - /volume1/data/frigate:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5050:5000"
      - "1935:1935" # RTMP feeds
    environment:
      - FRIGATE_RTSP_PASSWORD=xxxx
      - FRIGATE_MQTT_PASSWORD=xxxxx
```

**Camera Info:**
Nest Hello using Dropcam stream

**Comment by mjmeli** [comment]
Turns out the clips weren't working properly as they don't actually play

From the FAQ
>I have clips and snapshots in my clips folder, but I can't view them in the Web UI.#
This is usually caused one of two things:
>* The permissions on the parent folder don't have execute and nginx returns a 403 error you can see in the browser logs
**In this case, try mounting a volume to /media/frigate inside the container instead of /media/frigate/clips.

So the permissions on my NAS file system are all set up with chmod 777 on the frigate folder itself plus all containing folders.

I attached a console to the Frigate container and found that `/media/frigate` permissions were set as
`drwx------ 1 1026 users  30 Sep 23 12:21 frigate`

I ran `chmod 777 /media/frigate` and the issues instantly resolved.

Any reason why these permissions are not getting passed to the container?
